src/Illuminate/Auth/EloquentUserProvider.php

```diff
@@ -64,7 +64,9 @@ public function retrieveByToken($identifier, $token)
 
         $model = $model->where($model->getAuthIdentifierName(), $identifier)->first();
 
-        return $model && hash_equals($model->getRememberToken(), $token) ? $model : null;
+        $rememberToken = $model->getRememberToken();
```

Comment:
@Naxon What if `$model == null` here? There is a null-check for it two lines below, therefore I'm asking...

Reply:
This review makes sense. Why use `$model` to get rememberToken then right after it, check if the `$model` is `null`?

Reply:
Addressed in #21368.

Reply:
Addressed in #21368.